### PR TITLE
Verify user 

### PR DESCRIPTION
**Pull request by hochochoc**
- Verify consumer 
  - If exists, verify new input selfie with the reference by Compreface (similarity >= 0.95)
  - If not, add a reference selfie to Compreface
  - After all, return the result of the verification (if successfully, consumer get an access_token)

### CODE REVIEW

File: src/image/image.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@Resolver()
+export class ImageResolver {
+  @Mutation(() => Boolean)
+  async uploadFile(
+    @Args({ name: 'file', type: () => GraphQLUpload })
+    file: Promise<FileUpload>,
+  ) {
+    const { filename, mimetype, encoding, createReadStream } = await file;
+    console.log('attachment:', filename, mimetype, encoding);
```
Comment (jsw324) [inline src/image/image.resolver.ts:13]: let's not merge log statements into main

**Review by jsw324 — COMMENTED** [review]
If you can write a brief description of the PR, what it is supposed to do, if a query/mutation was added, what the signature of that may look like, etc. - that would be really helpful, especially when viewing the commit/PR history.

File: src/consumer/consumer.module.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Consumer } from 'consumer/entitites/consumer.entity';
```
Comment (imcvampire) [inline src/consumer/consumer.module.ts:3]: Within a module, you should use relative path

File: src/consumer/consumer.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      // verify consumer with selfie input
+    let matchSubjects 
+    try {
+      const response = await comprefaceService.verify(formData, consumer.selfie_uuid, {limit: 1})
+      matchSubjects = response
+    } catch (error) {
+      throw new BadRequestException(error);
+    };
+    
+    console.log(matchSubjects)
```
Comment (hochochoc) [inline src/consumer/consumer.resolver.ts:46]: remove console.log

File: src/consumer/consumer.resolver.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        email: consumer.email,
+        sub: consumer.id,
+      }),
+      expiresIn: this.configService.get<string>('auth.expiresIn'),
+    };
+  }
+
+  @Query(() => Consumer)
+  @UseGuards(GqlAuthGuard)
+  async cInfo(@CurrentUser() consumer: Consumer) {
```
Comment (imcvampire) [inline src/consumer/consumer.resolver.ts:95]:
```suggestion
  async customer(@CurrentUser() consumer: Consumer) {
```

**Pull request merged**